.gitignore

```diff
@@ -2,3 +2,4 @@
 .#*
 /target/
 Cargo.lock
+.*.sw*
```

Comment:
Huh?

Reply:
Oh. Oops. Automatic addition that gets rid of vim swap files. I can remove it (or leave it?)